hasura/migrations/default/1675459146542_create_activities_table/up.sql

```diff
@@ -0,0 +1,45 @@
+CREATE TABLE "public"."activities" (
+    "id" bigserial NOT NULL,
+    "circle_id" bigint,
+    "organization_id" bigint NOT NULL,
+    "actor_profile_id" bigint,
+    "target_profile_id" bigint,
+    "contribution_id" bigint,
+    "epoch_id" bigint,
+    "user_id" bigint,
+
+    "action" varchar(100) NOT NULL,
```

Comment:
this field needs stricter validation -- it should fail to insert if it isn't in a specified set of strings

Reply:
talked with graffe - holding off on this for now.